cos-nvidia-gpu-installer/installer.sh

```diff
@@ -47,13 +47,12 @@ BIN_OUTPUT_DIR="${BASE_DIR}/bin"
 KERNEL_SRC_DIR="/lakitu-kernel"
 
 NVIDIA_DRIVER_DIR="/nvidia"
-NVIDIA_DRIVER_VERSION="375.26"
+NVIDIA_DRIVER_VERSION="375.66"
```

Comment:
Given the lack of versioning in the driver installer, WDYT about keeping the driver version consistent with the one shipped as part of the CUDA 8 installer?